### PR TITLE
Upgrade airflow

### PR DESCRIPTION
**Pull request by rohit-dimagi**
https://dimagi-dev.atlassian.net/browse/IIO-923

### CODE REVIEW

**Comment by rohit-dimagi** [comment]
Bump @snopoke @calellowitz

**Comment by rohit-dimagi** [comment]
> This looks fine to me. How did the testing go?

I haven't tested it yet. there are some backfill jobs  running there from last 2 days.

File: airflow.cfg [hunk 1]
```diff
@@ -76,7 +76,7 @@ donot_pickle = False
 dagbag_import_timeout = 30
 
 # The class to use for running task instances in a subprocess
-task_runner = BashTaskRunner
+task_runner = StandardTaskRunner
```
Comment (calellowitz) [inline airflow.cfg:79]: does this change how the tasks are run?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
-tornado<5.0.0
-apache-airflow[celery, postgres, rabbitmq, crypto, statsd]==1.9.0
-gunicorn==19.4
+tornado<6.0.0,>=5.0.0
+apache-airflow[celery, postgres, rabbitmq, crypto, statsd]==1.10.11
+gunicorn<20.0,>=19.5.0
+marshmallow<2.20,>=2.18.0
+kombu<4.7,>=4.6.10
```
Comment (calellowitz) [inline requirements.txt:5]: where do the new requirements come from?

File: airflow.cfg (hunk 1 above)
Comment (rohit-dimagi) [reply airflow.cfg:79]: Bashrunner is now depreciated in favor of this in new airflow. we don't use this runner in icds env.

File: requirements.txt (hunk 2 above)
Comment (rohit-dimagi) [reply requirements.txt:5]: These came up while performing upgrade to 1.10.x

**Pull request merged**